tests/test_serving.py

```diff
@@ -129,12 +129,13 @@ def test_windows_get_args_for_reloading(monkeypatch, tmp_path):
 @pytest.mark.filterwarnings("ignore::pytest.PytestUnraisableExceptionWarning")
 @pytest.mark.parametrize("find", [_find_stat_paths, _find_watchdog_paths])
 def test_exclude_patterns(find):
-    # Imported paths under sys.prefix will be included by default.
+    # Get a path to exclude
     paths = find(set(), set())
-    assert any(p.startswith(sys.prefix) for p in paths)
```

Comment:
You can't remove this assert, otherwise we don't know if we're actually running a meaningful test. The path we exclude has to have files under it to exclude.

Reply:
Because the path I'm excluding is a parent directory of a entry in the unfiltered list
I thought that this would be enough to guarantee that we have a meaningful test?
I can add an assert for that (after its selected)